travis/test.sh

```diff
@@ -4,7 +4,7 @@ set -ex
 export PATH="$PWD/bin:$PATH"
 
 # analyze all the Dart code in the repo
-flutter analyze --flutter-repo --no-current-directory --no-current-package --congratulate
+flutter analyze --flutter-repo
```

Comment:
Yeah, `--congratulate ` is already on by default, and `--no-current-directory --no-current-package` won't have any effect when run from the root of the flutter repo.